wazuh/wazuh-kibana-app
# Fix error in save before exit prompt in configuration edit

Closes https://github.com/wazuh/wazuh-kibana-app/issues/3341

This PR fixes an issue where, if you tried to exit a configuration edit menu after saving, it would still ask you to save.

Testing:

- Go to `Management/Configuration` and click on `Edit Configuration`
- Make any changes
- Check that attempting to exit without saving shows the prompt
- Save
- Check that attempting to exit does no longer show the prompt
